tests/sles4sap/patterns.pm

```diff
@@ -67,6 +67,10 @@ sub run {
               unless ($output =~ /i.?\s+\|\spatterns-$pattern\s+\|\spackage\s\|\sRequired/);
         }
     }
+    elsif (check_var('SLE_PRODUCT', 'sles')) {
```

Comment:
I'm hesitating to add an extra `if get_var('HANA')` here for installing libatomic1 only in HANA scenario.

Reply:
I believe they should fix the pattern.

Reply:
But go ahead anyway hehe.  Just a note they should fix the pattern.

Reply:
If `libatomic1` installation is needed only by HANA, I do think it would be better to only install it when `get_var('HANA')`, because otherwise we are changing the environment unnecessarily.

Reply:
> If `libatomic1` installation is needed only by HANA, I do think it would be better to only install it when `get_var('HANA')`, because otherwise we are changing the environment unnecessarily.

That's what I thought, thanks for the confirmation.

Reply:
> I believe they should fix the pattern.

It's not a pattern issue: the `sap_hana` pattern doesn't exist on SLES, that why. It could be added in the `sap_server` pattern, but as it's only for HANA maybe it's not a good idea.